pkg/image/prune/imagepruner.go

```diff
@@ -190,6 +190,7 @@ func addImageStreamsToGraph(g graph.Graph, streams *imageapi.ImageStreamList, al
 
 		glog.V(4).Infof("Adding ImageStream %s/%s to graph", stream.Namespace, stream.Name)
 		isNode := imagegraph.EnsureImageStreamNode(g, stream)
+		imagegraph.EnsureAllImageStreamTagsNode(g, stream)
```

Comment:
Is that data used anywhere?

On Wed, Jul 1, 2015 at 9:31 AM, David Eads <notifications@github.com> wrote:

> In pkg/image/prune/imagepruner.go
> <https://github.com/openshift/origin/pull/3404#discussion_r33677747>:
>
> > @@ -190,6 +190,7 @@ func addImageStreamsToGraph(g graph.Graph, streams *imageapi.ImageStreamList, al
> >
> >  		glog.V(4).Infof("Adding ImageStream %s/%s to graph", stream.Namespace, stream.Name)
> >  		isNode := imagegraph.EnsureImageStreamNode(g, stream)
> > +		imagegraph.EnsureAllImageStreamTagsNode(g, stream)
>
>  What info is missing from the stream to be able to create a fully
> populated ISTNode?
>
> Image. The nested DockerImage has a lot of data.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3404/files#r33677747>.
>